packages/fxa-auth-server/lib/l10n/bindings.ts

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { existsSync, readFileSync } from 'fs';
+import { join } from 'path';
+
+export type TranslationOpts = {
+  basePath: string;
+};
+export type L10nOpts = {
+  translations: TranslationOpts;
+};
+
+export type LocalizerOpts = L10nOpts;
+
+export class LocalizerBindings {
+  readonly opts: LocalizerOpts;
+  constructor(opts?: LocalizerOpts) {
+    this.opts = Object.assign(
+      {
+        translations: {
```

Comment:
I changed this from `l10n` to `translations`. I felt `l10n` was a little ambiguous since the Localizer class now lives in an `l10n` directory.